dnf-behave-tests/dnf/plugins-core/enable-plugins-dashes.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Tests for enabling plugins containing dashes in names
+
+Background: Enable plugins
+  Given I do not disable plugins
+    And I use repository "dnf-ci-fedora"
```

Comment:
Here you should make sure, that the config-manager plugin is really available and enabled. Either by checking `dnf -v` output or by checking the exit code of `dnf config-manager --help` command.

Reply:
I agree, thanks.